sip-test-kit/src/main/java/de/ikor/sip/foundation/testkit/util/RegexUtil.java

```diff
@@ -47,4 +58,12 @@ public static String reformatEscapeCharacter(String str) {
     }
     return pattern;
   }
+
+  private static boolean areBothDefined(String actual, String expected) {
+    return isNotEmpty(actual) && isNotEmpty(expected);
+  }
+
+  private static boolean areBothEmpty(String actual, String expected) {
+    return isEmpty(expected) && isEmpty(actual);
+  }
```

Comment:
Test case is read from a file so I would expect some empty characters that can go through the isEmpty check
```
  public static boolean isEmpty(CharSequence cs) {
    return cs == null || cs.length() == 0;
  }
```
_isBlank_ seems like a better fit.